packages/playground/remote/src/lib/setup-fetch-network-transport.ts

```diff
@@ -72,7 +72,12 @@ export async function handleRequest(data: RequestData, fetchFn = fetch) {
 	try {
 		const fetchMethod = data.method || 'GET';
 		const fetchHeaders = data.headers || {};
-		if (fetchMethod == 'POST') {
+
+		const lowercasedHeaders = new Map<string, string>();
+		for (const [key, value] of Object.entries(fetchHeaders)) {
+			lowercasedHeaders.set(key.toLowerCase(), value);
+		}
+		if (fetchMethod == 'POST' && !lowercasedHeaders.has('content-type')) {
```

Comment:
```suggestion
		const hasContentTypeHeader = Object.keys(fetchHeaders).some(
			(name) => /content-type/i.test(name)
		);
		if (fetchMethod == 'POST' && !hasContentTypeHeader) {
```

nit: If we aren't going to do any more header lookups using `lowercasedHeaders`, maybe this would be more direct.

Reply:
> ```suggestion
> 
> 		const hasContentTypeHeader = Object.keys(fetchHeaders).some(
> 
> 			(name) => name.toLowerCase() === "content-type"
> 
> 		);
> 
> 		if (fetchMethod == 'POST' && !hasContentTypeHeader) {
> 
> ```

Reply:
> I like the code style, although the regexp would require also ^ and $ so I'll go with strict equality.

Ah, good point! I'm usually obsessive about the use of `^` and `$` but missed that.